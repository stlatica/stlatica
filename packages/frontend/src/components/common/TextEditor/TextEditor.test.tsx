// import { render } from "@testing-library/react";
// import React from "react";
// import { describe, expect, test } from "vitest";
import { describe, test } from "vitest";

// import { Story1 } from "./TextEditor.stories";

describe("snapshot", () => {
  test("snapshot", () => {
    // const { asFragment } = render(<Story1 />);
    // expect(asFragment()).toMatchSnapshot();
    true;
  });
});
