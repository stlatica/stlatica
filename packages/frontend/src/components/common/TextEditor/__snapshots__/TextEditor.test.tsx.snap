// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`snapshot > snapshot 1`] = `
<DocumentFragment>
  <div
    class="h-32 bg-gray-800 p-6 text-white"
  >
    <div
      class="relative h-full"
    >
      <textarea
        class="peer block h-full w-full appearance-none rounded-lg border border-white bg-transparent p-3 px-2.5 pb-2.5 pt-4 text-sm text-gray-800 focus:border-blue-600 focus:outline-none focus:ring-0 dark:border-gray-600 dark:text-white dark:focus:border-blue-500"
        id="story01"
        maxlength="16"
        placeholder=" "
      />
      <label
        class="absolute start-1 top-2 z-10 origin-[0] -translate-y-4 scale-75 bg-white px-2 text-sm text-gray-500 duration-300 peer-placeholder-shown:top-6 peer-placeholder-shown:-translate-y-1/2 peer-placeholder-shown:scale-100 peer-focus:top-2 peer-focus:-translate-y-4 peer-focus:scale-75 peer-focus:px-2 peer-focus:text-blue-600 rtl:peer-focus:left-auto rtl:peer-focus:translate-x-1/4 dark:bg-gray-800 dark:text-gray-400 peer-focus:dark:text-blue-500"
        for="story01"
      >
        Name
      </label>
    </div>
  </div>
</DocumentFragment>
`;
